scripts/monitor_oracle.py

```diff
@@ -0,0 +1,80 @@
+import os
+import requests
+import time
+from dotenv import load_dotenv
+from datetime import datetime
+import pytz
+
+load_dotenv()
+
+INTERNAL_DISCORD_WEBHOOK_URL = os.getenv("INTERNAL_DISCORD_WEBHOOK_URL")
+ALCHEMY_API_KEY = os.getenv("ALCHEMY_API_KEY")
+ORACLE_CONTRACT_ADDRESS = "0x81C1001e1621d05bE250814123CC81BBb244Cb07"
+
+ALCHEMY_URL = f"https://eth-mainnet.alchemyapi.io/v2/{ALCHEMY_API_KEY}"
+
+ORACLE_TIME_THRESHOLD = os.getenv("ORACLE_TIME_THRESHOLD", 24.5)  # Time threshold in hours
+
+class MonitorOracle:
+    def __init__(self, webhook_url):
+        self.webhook_url = webhook_url
+
+    def send_discord_alert(self, message):
+        """Send an alert message to the Discord channel and ping everyone."""
+        payload = {
+            "content": "@everyone",
```

Comment:
it's fine to ping everyone when there's an oracle outage, but probably not a good idea to ping everyone when there's a normal update so we don't get ping fatigue

Reply:
actually it seems like you're not sending a message when the oracle is updated, only when it's NOT updated, so you can disregard this